lib/js/dom.nim

```diff
@@ -1235,6 +1237,11 @@ proc slice*(e: Blob, startindex: int = 0, endindex: int = e.size, contentType: c
 # Performance "methods"
 proc now*(p: Performance): float
 
+# Selection "methods"
+proc removeAllRanges*(s: Selection)
+converter toString*(s: Selection): cstring
```

Comment:
Maybe add a `{.deprecated.}` pragma here (not sure if that works, but should)

Reply:
I don't think it should be deprecated because its still a valid method that you can and will use.

I tried {.deprecated.} but now it just warns for valid calls.